### DISCUSSION TITLE
platform error occurs when try to install offline

### DISCUSSION THREAD

**Discussion by CoolJY** [sfu-db/connector-x#207] [Q&A]
I downloaded connectorx-0.2.2-cp37-cp37m-manylinux2014_x86_64.whl file and try to install it on my server.
My server OS is redhat8 and python version is 3.7.
I think there are no problems about platform and version.
But it did not work.

message : 
connectorx-0.2.2-cp37-cp37m-manylinux2014_x86_64.whl is not a supported wheel on this platform.

What's wrong?

**Comment by CoolJY** [comment]
0.2.2 and 0.2.3 have same problem.

**Reply by wangxiaoying** [reply]
Hi @CoolJY , I don't have a red hat instance so I cannot test to see whether it is the reason (the wheel can be installed on my local ubuntu instance using Python 3.7.). Another possible reason is the architecture (x86_64).

You can also build the wheel file manually as follow:

1. Configure environment. The instruction is [here](https://github.com/sfu-db/connector-x/blob/main/CONTRIBUTING.md#install-tools-and-dependencies).
2. Build connectorx. Run command `just ci-build-python-extention ci-build-python-wheel ci-rename-wheel` to generate a wheel file under directory `connectorx-python/dist/`.

**Reply by CoolJY** [reply]
@wangxiaoying Thank you for reply. But my environment is offline. I installed many x86_64 wheel files on my server, so it may not be the reason..
